Formula/hssp.rb

```diff
@@ -0,0 +1,35 @@
+class Hssp < Formula
+  # cite Touw_2015: "https://doi.org/10.1093/nar/gku1028"
+  desc "Create HSSP files"
+  homepage "https://github.com/cmbi/hssp"
+  url "https://github.com/cmbi/hssp/archive/3.1.5.tar.gz"
+  sha256 "9462608ce6b5b92f13a3a8d94b780d85a3cac68ab38449116193754cc22dc5d0"
+
+  bottle do
+    root_url "https://linuxbrew.bintray.com/bottles-bio"
+    cellar :any
+    sha256 "c0833e3e3db899b28dd9515e1d3ffb791528d2934d25226dbf4eaf0f0428b34f" => :sierra
+    sha256 "890f4a0285987df828d271ad333b597181358c75b65026d94dccf3d7c6d49531" => :x86_64_linux
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "boost"
```

Comment:
Is there run-time linkage to `boost`? What's the output of `brew linkage boost`?